### PR TITLE
refactor(frontend): rename stores regarding ETH transactions

### PR DESCRIPTION
**Pull request by AntonioVentilii**
# Motivation

To be less confusing, since we have now more than 2 networks, we rename all the Ethereum-related transactions stores with the prefix `Eth-`.

### CODE REVIEW

File: src/frontend/src/eth/stores/eth-transactions.store.ts [hunk 1]
```diff
@@ -41,4 +41,4 @@ const initTransactionsStore = (): TransactionsStore => {
 	};
 };
 
-export const transactionsStore = initTransactionsStore();
+export const ethTransactionsStore = initTransactionsStore();
```
Comment (peterpeterparker) [inline src/frontend/src/eth/stores/eth-transactions.store.ts:44]: You will rename the rest of the module, such as its interfaces, in a separate PR?

**Review by peterpeterparker — APPROVED** [review]
LGTm, thx

File: src/frontend/src/eth/stores/eth-transactions.store.ts (hunk 1 above)
Comment (AntonioVentilii) [reply src/frontend/src/eth/stores/eth-transactions.store.ts:44]: correct, I am doing it slowly, just to avoid any conflicts with other PRs

**Review by peterpeterparker — APPROVED** [review]
LGTM, thx

**Comment by AntonioVentilii** [comment]
[OISY-130]

**Pull request merged**